data/api/identity/v2_pubkey.yaml

```diff
@@ -105,7 +105,7 @@ paths:
         - in: query
           name: public_key
           required: true
-          description: The unpadded base64-encoded public key to check.
+          description: The unpadded standard or URL-safe base64-encoded public key to check.
```

Comment:
likewise, this is a simplification. The choice of encoding is not up to the caller; rather, the caller has to use the same encoding as is in the `m.room.third_party_invite` url.

Reply:
I tried to clarify this by saying to use the exact same string.